### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by bytesleo** [Schweinepriester/achievement-testarea#39] [Q&A] [answered]
Hi guys, I'm here to show you a nice repository so you can help me with your star, thanks!  🙌

[https://github.com/kevoj/angular-editor-fabric-js](https://github.com/kevoj/angular-editor-fabric-js)

**Comment by zoelesv** [comment, marked as answer]
Cool repo! I gave it a ⭐️. Check out my repo and help me get my first star-truck as well! https://github.com/zoelesv/Smathchat

**Comment by Rantoryu** [comment]
@kevoj @zoelesv I've starred your repos ⭐️. If You can give me star here too I would appreciate it: https://github.com/Rantoryu/Personal_Website

**Reply by zoelesv** [reply]
Done!

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.
